components/user_settings/advanced/user_settings_advanced.tsx

```diff
@@ -168,19 +196,19 @@ export default class AdvancedSettingsDisplay extends React.PureComponent {
         }
     }
 
-    handleShowDeactivateAccountModal = () => {
+    handleShowDeactivateAccountModal = (): void => {
         this.setState({
             showDeactivateAccountModal: true,
         });
     }
 
-    handleHideDeactivateAccountModal = () => {
+    handleHideDeactivateAccountModal = (): void => {
```

Comment:
do we really need to add the return type?

I don't think we do that in a lot of places and we actually had a discussion a while ago where we decided to let TS infer the return type (at least for simple things like this)

Reply:
In the previous PR reviews, I received a feedback to put return types for functions. For Future PRs and simple functions I can let TS info the return type.

Reply:
Linking the discussion for everyone's visibility: https://github.com/mattermost/mattermost-webapp/pull/10661#discussion_r914934817

Reply:
I'll bring this up in the next Web Guild meeting to see if anyone has any strong opinions either way. Personally, I like them for functions that are being exported for use outside the web app since it means that we're less likely to accidentally introduce breaking changes, but inside the app, I don't particularly care one way or the other. Either way, we should try to get on the same page though to avoid confusing people during PR reviews with conflicting info